doc/development_process/gh_labels.rst

```diff
@@ -0,0 +1,138 @@
+               the bug or pull request.
+=============  ===============================================================
+
+An area maintainer should be able to filter by an area label and
+find all issues and PRs which relate to that area.
+
+Platform
+*********
+
+=============  ===============================================================
+Labels         ``Platform:*``
+Applicable to  PRs  and issues
+Description    An issue or PR which affects only a particular platform
+=============  ===============================================================
+
+To be discussed in a meeting
+****************************
+
+=============  ===============================================================
+Labels         ``API``, ``dev-review``, ``TSC``
+Applicable to  PRs  and issues
+Description    The issue is to be discussed in the following
+               `API/dev-review/TSC meeting`_ if time permits
+=============  ===============================================================
+
+.. _`API/dev-review/TSC meeting`: https://github.com/zephyrproject-rtos/zephyr/wiki/Zephyr-Committee-and-Working-Group-Meetings
+
+Minimum PR review time
+**********************
+
+=============  ===============================================================
+Labels         ``Hot Fix``, ``Trivial PR``, ``Maintainer``,
+               ``Security Review``, ``TSC``
+Applicable to  PRs only
+Description    Depending on the PR complexity, an indication of how long a merge
+               should be held to ensure proper review. See
+               :ref:`review process <review_time>`
+=============  ===============================================================
+
+Issue priority labels
+*********************
+
+=============  ===============================================================
+Labels         ``priority:{high|medium|low}``
+Applicable to  Issues only
+Description    To classify the impact and importance of a bug or feature
+=============  ===============================================================
+
+Note: Issue priorities are generally set or changed during the bug-triage or TSC
+meetings.
+
+Miscellaneous labels
+********************
+
+For both PRs and issues
+=======================
+
++------------------------+-----------------------------------------------------+
+|``Bug``                 | The issue is a bug, or the PR is fixing a bug       |
++------------------------+-----------------------------------------------------+
+|``Coverity``            | A Coverity detected issue or its fix                |
++------------------------+-----------------------------------------------------+
+|``Waiting for response``| The Zephyr developers are waiting for the submitter |
+|                        | to respond to a question, or address an issue.      |
++------------------------+-----------------------------------------------------+
+|``RFC``                 | The author would like input from the community. For |
```

Comment:
Should we consider deprecating this label in favor of using draft pull requests only? https://github.blog/2019-02-14-introducing-draft-pull-requests/

I see below that `This PR should not be merged (Do Not Merge).  For work in progress, GitHub "draft" PRs are preferred"` is listed next to DNM. I also see `In progress` is a label which can apply to PRs that seems very similar.

As a reader, I'm a bit confused between the following labels for pull requests, which all seem to be related:

 - RFC
- In progress
- DNM
- GitHub "Draft" PRs

Is there an opportunity to consolidate these into just one label (or perhaps just use draft PRs)?

Reply:
Maybe, but not trivially right now.
(Note that I'm not trying to fix all issues related to our way of working with this PR)
I'm hoping to start by document what we have, and at the same time, clean what is trivial to clean.
If we start by going further in this same PR it will just balloon out of proportion.
=> If you don't mind, let's take that discussion separately from this PR.

Reply:
Sure